test/apiv2/10-images.at

```diff
@@ -440,4 +440,18 @@ t GET "libpod/events?stream=false&since=$START"  200  \
   .Actor.Attributes.name="localhost:5000/idonotexist" \
   .Actor.Attributes.error~".*connection refused"
 
+# test empty RepoTags and RepoDigests is an empty array
+IIDFILE=$(mktemp)
+podman image build --iidfile $IIDFILE -<<EOF
+from alpine
+RUN :
+EOF
+
+t GET images/json 200 \
+  .[2].RepoTags=[] \
+  .[2].RepoDigests=[]
+
+podman rm -f $(cat $IIDFILE)
```

Comment:
This runs rm not rmi so this will not work and because you used `-f` it won't error if the arg does not exists.

Also FYI, you don't need cat `$(< $IIDFILE)` works the same without the extra process exec.

Reply:
ah! `rmi`, my bad